package.json

```diff
@@ -21,6 +21,10 @@
     }
   },
   "eslintConfig": {
+    "env": {
+      "node": true,
```

Comment:
@existentialcoder : should we set `browser: true` ? Our package can be useful for any frontend or backend js app, isn't it?